paasta_tools/paasta_cli/utils.py

```diff
@@ -540,3 +540,45 @@ def get_jenkins_build_output_url():
     if build_output:
         build_output = build_output + 'console'
     return build_output
+
+
+class PaastaValidateMessages:
```

Comment:
also, why are these in a class? these can just be constants for the validate module.